src/test/pages/Groups.test.tsx

```diff
@@ -156,12 +157,12 @@ describe("Groups", () => {
         let correctDeleteRequests = 0;
         renderTestEnvironment({
             extraEndpoints: [
-                rest.delete(API_PATH + "/groups/:groupId", async (req, res, ctx) => {
-                    const {groupId} = req.params;
+                http.delete(API_PATH + "/groups/:groupId", async ({params}) => {
+                    const {groupId} = params;
                     if (parseInt(groupId as string) === groupToDelete.id) {
                         correctDeleteRequests++;
                     }
-                    return res(ctx.status(204));
+                    return HttpResponse.json(null, {status: 200,});
```

Comment:
Strange that `204` in here doesn't work, must be a bug. This seems to work though:

```suggestion
                    return new HttpResponse(null, {status: 204, headers: {'Content-Type': 'application/json'}});
```

Reply:
Ah, nice. It even works without the header being explicitly set; presumably there's a bug with the JSON factory method in particular for empty objects.